levenshtein_test.go

```diff
@@ -120,3 +120,45 @@ func BenchmarkAll(b *testing.B) {
 	}
 	sink = tmp
 }
+
+// Fuzzing
+// ----------------------------------------------
+
+func FuzzComputeDistanceDifferent(f *testing.F) {
+	testcases := []struct{ a, b string }{
+		{"levenshtein", "frankenstein"},
+		{"resumé and café", "resumés and cafés"},
+		{"Hafþór Júlíus Björnsson", "Hafþor Julius Bjornsson"},
+		{"།་གམ་འས་པ་་མ།", "།་གམའས་པ་་མ"},
+	}
+	for _, tc := range testcases {
+		f.Add(tc.a, tc.b)
+	}
+	f.Fuzz(func(t *testing.T, a, b string) {
+		n := agnivade.ComputeDistance(a, b)
+		if n < 0 {
+			t.Errorf("Distance can not be negative: %d, a: %q, b: %q", n, a, b)
+		}
+		if n > len(a)+len(b) {
+			t.Errorf("Distance can not be greater than sum of lengths of a and b: %d, a: %q, b: %q", n, a, b)
+		}
```

Comment:
I was hoping to check more properties, but I could only come up with this list. Is there more to check?

Reply:
This looks good Ben!